File: 6-kyu/is-a-number-prime-/python/solution.py
from math import sqrt

def is_prime(num):
    if num < 2:
        return False
    if num == 2:
        return True
    if num % 2 == 0:
        return False
    i = 3
    while i <= sqrt(num):
        if num % i == 0:
            return False
        i += 2
    return True

File: 6-kyu/is-a-number-prime-/javascript/solution.js
const isPrime = (num) => {
  if (num < 2) {
    return false;
  } else if (num === 2) {
    return true;
  } else if (num % 2 === 0) {
    return false;
  }
  for (let i = 3; i <= num ** 0.5; i += 2) {
    if (num % i === 0) {
      return false;
    }
  }
  return true;
}